addon-test-support/@ember/test-helpers/dom/trigger-key-event.js

```diff
@@ -0,0 +1,23 @@
+import { merge } from '@ember/polyfills';
+import triggerEvent from './trigger-event';
+
+/**
+ * @public
+ * @param selector
+ * @param type
+ * @param keyCode
+ * @param modifiers
+ * @return {*}
+ */
+export function keyEvent(
+  selectorOrElement,
+  type,
+  keyCode,
+  modifiers = { ctrlKey: false, altKey: false, shiftKey: false, metaKey: false }
+) {
+  return triggerEvent(
+    selectorOrElement,
+    type,
+    merge({ keyCode, which: keyCode, key: keyCode }, modifiers)
```

Comment:
can we extract that into an `options` variable so that the `triggerEvent()` call stays single-line?

Reply:
Yep, that would look much better indeed.